src/metabase/driver.clj

```diff
@@ -532,7 +538,10 @@
     :connection-impersonation-requires-role
 
     ;; Does the driver require specifying a collection (table) for native queries? (mongo)
-    :native-requires-specified-collection})
+    :native-requires-specified-collection
+
+    ;; Does the driver support column(s) indexing
```

Comment:
I wonder if this should read: 
`Does the driver support column(s) support storing index info`

Implementing the multimethods to sync index info should be separate from whether indices are supported on the database or not.

Also wondering whether the feature should be called `:index-info`

Reply:
yea index-info sounds better to me, renamed in https://github.com/metabase/metabase/pull/36462/commits/07847b53438d0b22d3f058ace91904b64953dbc7